go-to-libraries.md

```diff
@@ -37,3 +37,9 @@ library doesn't appear to be very well maintained or documented and there are
 places where the documentation doesn't match the library or the library doesn't
 behave in documented ways.  Expect to end up reading the code some if you
 choose this for anything very complex.
+
+### [`cached-property`](https://pypi.org/project/cached-property/)
+
+@piper
```

Comment:
not sure why this is here

Reply:
Yeah, I figure it's good to know who was championing something.  In theory git blame works for this but that's bad UX.

Reply:
`@piper suggests:`
or
`@piper recommends:`
to avoid future questions of the same type :)